Library/Homebrew/os/mac/xcode.rb

```diff
@@ -234,6 +234,30 @@ def detect_version_from_clang_version
       def default_prefix?
         prefix.to_s == "/Applications/Xcode.app/Contents/Developer"
       end
+
+      sig { params(swift_version: String).returns(String) }
+      def minimum_version_for_swift(swift_version)
```

Comment:
Does the CLT provide a Swift compiler?

Reply:
Yes, but it appears to be missing the syntax parser library that ~3 formulae use. This is kind of annoying given that Xcode.app isn't in a fixed path so I'll look at that a bit closer.

We could therefore relax the requirement and allow the CLT in cases where the syntax parser library is not used (though such a requirement for a minimum CLT version doesn't currently exist).

Reply:
> We could therefore relax the requirement and allow the CLT in cases where the syntax parser library is not used (though such a requirement for a minimum CLT version doesn't currently exist).

Perhaps could be an additional `:xcode` tag for the requirement? Would be nice to use the CLT when it's possible to do so.